src/lib/Scenes/Partner/Components/PartnerOverview.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, Sans, Spacer } from "@artsy/palette"
+import { PartnerOverview_partner } from "__generated__/PartnerOverview_partner.graphql"
+import { ReadMore } from "lib/Components/ReadMore"
+import { get } from "lib/utils/get"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+const textLimit = 50
```

Comment:
Do we want to increase this text limit for iPad? `hardware.ts` also has something for this, but the values might not match the design you have in mind.